content/bibs/pubs.bib

```diff
@@ -1522,19 +1556,34 @@ @article {Rose2021
     keywords = {peer-reviewed}
 }
 
+@article{bridgeford2021eliminating,
+  title={Eliminating accidental deviations to minimize generalization error and maximize replicability: Applications in connectomics and genomics},
+  author={Bridgeford, Eric W and Wang, Shangsi and Wang, Zeyi and Xu, Ting and Craddock, Cameron and Dey, Jayanta and Kiar, Gregory and Gray-Roncal, William and Colantuoni, Carlo and Douville, Christopher and others},
+  author+an = {1=trainee; 6=trainee; 17=highlight},
+  journal={PLoS computational biology},
+  volume={17},
+  number={9},
+  pages={e1009279},
+  year={2021},
+  url = {https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009279},
+  publisher={Public Library of Science San Francisco, CA USA},
+  keywords = {peer-reviewed}
+}
+
 @article {Bridgeford802629,
 	author = {Bridgeford, Eric W. and Wang, Shangsi and Yang, Zhi and Wang, Zeyi and Xu, Ting and Craddock, Cameron and Dey, Jayanta and Kiar, Gregory and Gray-Roncal, William and Coulantoni, Carlo and Douville, Christopher and Priebe, Carey E. and Caffo, Brian and Milham, Michael and Zuo, Xi-Nian and Vogelstein, Joshua T.},
-	author+an = {1=trainee; 16=highlight},
```

Comment:
Why delete information here? I saw there's a peer-reviewed version of the article, so this entry could be completely replaced?

Reply:
This one is the remnant of the duplicate from which I copied some information from. This is now deleted! Thank you so much for all these correction, @PSSF23